frontend/src/scenes/insights/TrendLegend.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Table } from 'antd'
+import { useActions, useValues } from 'kea'
+import { IndexedTrendResult, trendsLogic } from './trendsLogic'
+import { PHCheckbox } from 'lib/components/PHCheckbox'
+import { getChartColors } from 'lib/colors'
+
+export function TrendLegend(): JSX.Element {
```

Comment:
Q: Why do we show the values here? It feels like it duplicates the graph, makes the table view unneeded and makes the whole thing too wide to fit on the side.

Reply:
I'm indifferent. I went this route because putting the legend on the right side of the trend graph would add a lot of horizontal density. By putting the legend below, there's a lot of space so I figured we could just fill up the exact numbers for easier comparison. 

tagging @timgl @corywatilo for drive by thoughts
<img width="1249" alt="Screen Shot 2021-02-24 at 10 32 42 AM" src="https://user-images.githubusercontent.com/13127476/109024521-db7f0b00-768b-11eb-957d-c2236e81597d.png">

Reply:
I like the table, especially when breaking down it's nice to get the numbers as well. When I'm using amplitude I found myself looking at the table below a lot to get the precise numbers.

I think we should add something like this too, definitely for dashboards at least.
![image](https://user-images.githubusercontent.com/1727427/109052223-a8646800-76db-11eb-9d15-4fdc36c9c5d4.png)

I'm not super convinced about the <hr> and the spacing. Doesn't look quite right to me. Also i think the table should hit the sides?

(also made a quick commit to make the table small)